src/components/CardList.js

```diff
@@ -0,0 +1,34 @@
+import Card from './Card';
+import styled from 'styled-components';
+import { vocabulary } from '../db';
+
+export default function CardList() {
+  return (
+    <>
+      <Wrapper>
+        <AllCards role="list" aria-label="vocabulary cards" rows={3}>
```

Comment:
Maybe you can choose a better name for the list here. For example `StyledList`

Reply:
right, it sounds better :)